pipeline/outputs/stackdriver.md

```diff
@@ -184,7 +184,17 @@ Do following check:
 * If the log entry does not contain the local_resource_id field, does the tag of the log match for format?
 *   If tag_prefix is configured, does the prefix of tag specified in the input plugin match the tag_prefix?
 
-    **Other implementations**
+### caught signal (SIGSEGV)
```

Comment:
I would recommend changing this heading, because `caught signal (SIGSEGV)` can be any manner of different things. How about:
```suggestion
### Crashing with default number of `Workers`
```